data/adapters/storage.py

```diff
@@ -0,0 +1,126 @@
+import yaml
```

Comment:
this is the old __init__.py with only the methods we currently need, we will continue to modify and refactor as more Clickhouse features get added.